.github/workflows/release.yml

```diff
@@ -10,10 +10,10 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 20
-      - uses: actions/setup-java@v3
+      - uses: actions/setup-java@v4
         with:
           distribution: "temurin"
-          java-version: 11
```

Comment:
so the plugin will be built with java 17 and published. will a user be able to install it in IntelliJ 2022.1?

Reply:
For compatibility with 2022.1, we need to produce Java 11 bytecode. However, this is [configured](https://github.com/sourcegraph/jetbrains/blob/main/gradle.properties#L18) in a different place.